scripts/mkcloud

```diff
@@ -1126,20 +1126,16 @@ Optional
         an intrinsic default is chosen. Only possible with SLE12 node.
     want_all_debug=1 (default='')
         Enable debug level logging in barclamp proposals for all proposals.
-    want_magnum=1 (default=0)
-        It's not deployed by default. Set to 1 to deploy magnum.
-    want_monasca=1 (default=0)
-      Set to 1 to deploy monasca (Cloud7+ only)
-    want_barbican=0 (default=1)
-        Deployed by default. Set to 0 to prevent barbican from being deployed.
-    want_sahara=0 (default=1)
-        Deployed by default. Set to 0 to prevent sahara from being deployed.
-    want_murano=1 (default=0)
-        It's not deployed by default. Set to 1 to deploy murano.
-    want_trove=0 (default=1)
-        Deployed by default. Set to 0 to prevent trove from being deployed.
-    want_aodh=1 (default=1)
-        Deployed by default. Set to 0 to prevent aodh from being deployed.
+    want_\$PROPOSAL_proposal=[01]
+        Set to 0 to prevent \$PROPOSAL from being deployed. e.g.
+        want_aodh_proposal=0
```

Comment:
@aplanas 
What I wanted to highlight was this:
> Set to 0 to **prevent** $PROPOSAL from being deployed. e.g.

and

> Set to 1 to **enable** $PROPOSAL to be deployed (**that are default off**). e.g.

Maybe that makes it clearer.